node/clients/search.ts

```diff
@@ -74,65 +82,84 @@ export class Search extends AppClient {
 
   public product = (slug: string) =>
     this.get<SearchProduct[]>(
-      `/pub/products/search/${this.searchEncodeURI(slug && slug.toLowerCase())}/p`,
+      this.addCompleteSpecifications(`/pub/products/search/${this.searchEncodeURI(slug && slug.toLowerCase())}/p`),
```

Comment:
I know this is not related to this PR, but do we need that short circuiting? `slug &&`? 
```suggestion
      this.addCompleteSpecifications(`/pub/products/search/${this.searchEncodeURI(slug.toLowerCase())}/p`),
```

Reply:
I sincerely don't know, I'll leave it be